### ISSUE TITLE
[enhancement 1/2] add freed up space

### ISSUE THREAD

**Issue by mattia-b89** [graysky2/profile-sync-daemon#190]
when I run `psd c` I'd like to see the overall space freed up with this operation
this is just an idea. thank you for your big work!

**Comment by graysky2** [comment]
Currently, running `psd p` gives you this info.  Is that insufficient for you?

```% psd p
Profile-sync-daemon v6.29 on Arch Linux.
...
 recovery dirs:   2 <- delete with the c option
  dir path/size:  /home/facade/.config/chromium-backup-crashrecovery-20150117_171359 (92M)
  dir path/size:  /home/facade/.config/chromium-backup-crashrecovery-20150119_112204 (93M)
...
```

**Comment by mattia-b89** [comment]
Taking into account that: 1. Issuing "psd d" on a terminal is "hard" to
read the size since lines are wrapped 2. Could happen (my case) you have
many (more than ten) backup.
It's hard to get the sum.

I can't answer you actual information are not sufficient but the sum, that
is a recap of the all sizes, would ease the life

powered by CM

**Comment by graysky2** [comment]
Your xterm should be wider if that is word wrapping;  I have implemented #191 by you so you should be able to limit to whatever number you want now to simplify the display.
